frontend/src/js/ui-components/CurrencyInput.tsx

```diff
@@ -48,10 +48,12 @@ const CurrencyInput: FC<PropsT> = ({
       setNumberFormatValue("");
     }
   }, [value]);
-
   return (
     <SxNumberFormat
-      {...currencyConfig}
+      thousandSeparator={currencyConfig?.thousandSeparator}
+      decimalSeparator={currencyConfig?.decimalSeparator}
+      decimalScale={currencyConfig?.decimalScale}
+      suffix={" " + currencyConfig?.unit}
```

Comment:
Easier like this:
```suggestion
     {...currencyConfig}
      suffix={" " + currencyConfig.unit}
```

( I think we can safely make `currencyConfig` **non-optional** in `CurrencyInput` )